comp/otelcol/ddflareextension/impl/go.mod

```
@@ -69,6 +69,7 @@ require (
 	github.com/DataDog/datadog-agent/comp/otelcol/configstore/def v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/comp/otelcol/configstore/impl v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/comp/otelcol/ddflareextension/def v0.0.0-00010101000000-000000000000
+	github.com/DataDog/datadog-agent/pkg/version v0.0.0-00010101000000-000000000000
```

we [import v0.57.0 here](https://github.com/DataDog/datadog-agent/blob/5045eb2f5e58c40f66f585b405f83e42cb2d98fb/comp/otelcol/converter/impl/go.mod#L76) and [here](https://github.com/DataDog/datadog-agent/blob/5045eb2f5e58c40f66f585b405f83e42cb2d98fb/go.mod#L707); does it make sense to make this consistent across the repo?
```suggestion
	github.com/DataDog/datadog-agent/pkg/version v0.57.0
```